core/vm/actor/cron_actor.cpp

```diff
@@ -6,24 +6,30 @@
 #include "vm/actor/cron_actor.hpp"
 
 namespace fc::vm::actor {
+
+  using vm::actor::MethodNumber;
+  using vm::runtime::InvocationOutput;
+  using vm::runtime::Runtime;
+
   std::vector<CronTableEntry> CronActor::entries = {
       {kStoragePowerAddress, SpaMethods::CHECK_PROOF_SUBMISSIONS}};
 
-  outcome::result<Buffer> CronActor::epochTick(
+  outcome::result<InvocationOutput> CronActor::epochTick(
       const Actor &actor,
-      vm::VMContext &vmctx,
-      gsl::span<const uint8_t> params) {
-    if (!(vmctx.message().from == kCronAddress)) {
+      const std::shared_ptr<Runtime> &runtime,
+      const MethodParams &params) {
+    if (!(runtime->getMessage()->from == kCronAddress)) {
       return WRONG_CALL;
     }
 
     for (const auto &entry : entries) {
-      OUTCOME_TRY(vmctx.send(entry.to_addr, entry.method_num, BigInt(0), {}));
+      OUTCOME_TRY(
+          runtime->send(entry.to_addr, entry.method_num, {}, BigInt(0)));
     }
     return outcome::success();
   }
 
   ActorExports CronActor::exports = {
-      {2, ActorMethod(CronActor::epochTick)},
+      {MethodNumber{2}, ActorMethod(CronActor::epochTick)},
```

Comment:
no magic numbers should present. pls consider making a enum and accessing method number by name.

Reply:
ok